java/maven.embedder/nbproject/project.xml

```diff
@@ -319,9 +319,14 @@
             <class-path-extension>
                 <runtime-relative-path>../maven/lib/httpcore-4.4.16.jar</runtime-relative-path>
             </class-path-extension>
+            <!--
+            disable Maven's ANSI CLI decorations
+            maven-slf4j-provider injects MavenSimpleLogger which decorates the output for terminal use
+            this causes problems in log files, tests etc which don't expect or cant display them properly
             <class-path-extension>
                 <runtime-relative-path>../maven/lib/jansi-2.4.0.jar</runtime-relative-path>
             </class-path-extension>
+            -->
```

Comment:
thats likely the best way to be 100% sure that it is working:
here is the maven code:

https://github.com/apache/maven/blob/3c65873b584945c034aeb4ba51a5793c6582b19d/maven-embedder/src/main/java/org/apache/maven/cli/jansi/MessageUtils.java#L52-L61

Reply:
Great !!